src/api/swapTypes.ts

```diff
@@ -0,0 +1,50 @@
+import { toBitcoin } from "satoshi-bitcoin-ts";
+import { fromWei } from "web3-utils";
+
+export interface Asset {
+  name: string;
+  quantity: string;
+  token_contract?: string;
+}
+
+export function toMainUnit(asset: Asset) {
+  switch (asset.name) {
+    case "ether":
+      return fromWei(asset.quantity, "ether") + " ETH";
```

Comment:
🔧 The result of the conversion functions for the different assets produces different results when displayed in the UI (which is a little weird). We should probably unify the result after the conversion so it is displayed the same way.
Currently when sending 1 Satoshi to 1 Wei:
Bitcoin:
```
Selling 1e-8 BTC on bitcoin testnet
```
Ether:
```
Buying 0.000000000000000001 ETH on ethereum ropsten
```

Reply:
Good point! I never noticed because we always try to test with 1BTC for 1ETH.